geomesa-kafka/geomesa-kafka-datastore/geomesa-kafka-10-datastore/src/main/scala/org/locationtech/geomesa/kafka10/KafkaConsumerFeatureSource.scala

```diff
@@ -83,6 +84,10 @@ object KafkaConsumerFeatureSourceFactory {
       Option(KafkaDataStoreFactoryParams.CLEANUP_LIVE_CACHE.lookUp(params).asInstanceOf[Boolean]).getOrElse(false)
     }
 
+    val cacheCleanUpPeriod: Long = {
+      Suffixes.Time.millis(KafkaDataStoreFactoryParams.CACHE_CLEANUP_PERIOD.lookUp(params).asInstanceOf[String]).getOrElse(1000L)
```

Comment:
Could you change the `.getOrElse` to be 10000 instead of 1000? That way we keep consistency with the old code and the default param value. (in 3 places)